fugue-aws/src/main/java/com/symphony/oss/fugue/aws/kv/table/AbstractDynamoDbKvTable.java

```diff
@@ -2043,7 +2041,7 @@ void consume(Item item, ITraceContext trace)
     protected final AmazonDynamoDBClientBuilder amazonDynamoDBClientBuilder_;
 
     protected String              region_;
-    protected int                 payloadLimit_           = MAX_RECORD_SIZE;
+    protected Integer             payloadLimit_           = MAX_RECORD_SIZE;
     protected boolean             validate_               = true;
     protected boolean             enableSecondaryStorage_ = false;
```

Comment:
Is this dead now? Should we delete it?

Reply:
no it's used,  but in order to check if the value is not set i check if it's null